roles/openshift_serviceaccounts/tasks/main.yml

```diff
@@ -1,26 +1,33 @@
+- name: tmp dir for openshift
+  file:
+    path: /tmp/openshift
+    state: directory
+    owner: root
+    mode: 700
+
```

Comment:
Man, never really looked at this role before...  This really should be a module handling this rather than scripted commands, but that is a completely different issue :)